<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaymentTab</class>
 <widget class="QWidget" name="PaymentTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>740</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Payment Information</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Booking:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="bookingCombo"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Amount:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="amountInput">
        <property name="prefix">
         <string>$ </string>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Payment Method:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="paymentMethodCombo">
        <item>
         <property name="text">
          <string>Credit Card</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Debit Card</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cash</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mobile Payment</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Payment Status:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="paymentStatusCombo">
        <item>
         <property name="text">
          <string>Completed</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Pending</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Failed</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Refunded</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="paymentTable">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Payment ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Customer</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Movie</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Amount</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Payment Method</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Status</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Payment Date</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addPaymentBtn">
       <property name="text">
        <string>Add Payment</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updatePaymentBtn">
       <property name="text">
        <string>Update Payment</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deletePaymentBtn">
       <property name="text">
        <string>Delete Payment</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="refreshPaymentBtn">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
